modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -56,6 +58,14 @@ def __init__(self, object_id, length=None, width=None, ip=None, call_queue=None)
         self._length_cache = length
         self._width_cache = width
         self._ip_cache = ip
+        self._identity = uuid.uuid4().hex[8:]
+
+        logger = get_logger()
+        logger.debug(
+            "Partition OID: {}, Height: {}, Width: {}".format(
```

Comment:
could you also add the object `_identity` here? It would allow us to reference the partition `_identity` logged in `get()`, `apply()`, etc.

Reply:
it may also be helpful to log the `_ip_cache`. I think @RehanSD had mentioned it might be useful, could you comment on this?

Reply:
Yes, will log these as well. Object `_identity` will actually be better than the partition's `oid` so will change that.